pep-0501.txt

```diff
@@ -347,23 +421,23 @@ specific proposal in this PEP is designed to make it straightforward to write
 use case specific renderers that take care of quoting interpolated values
 appropriately for the relevant security context::
 
-    runquery(sql(i"SELECT {column} FROM {table};"))
-    runcommand(sh(i"cat {filename}"))
-    return_response(html(i"<html><body>{response.body}</body></html>"))
+    runquery(sql(t"SELECT {column} FROM {table} WHERE column={value};"))
+    runcommand(sh(t"cat {filename}"))
+    return_response(html(t"<html><body>{response.body}</body></html>"))
 
-This PEP does not cover adding such renderers to the standard library
-immediately, but rather proposes to ensure that they can be readily provided by
+This PEP does not cover adding all such renderers to the standard library
+immediately (though one for shell escaping is proposed), but rather proposes to ensure that they can be readily provided by
 third party libraries, and potentially incorporated into the standard library
 at a later date.
 
-For example, a renderer that aimed to offer a POSIX shell style experience for
+It is proposed that a renderer is included in the `shlex` module, aimed to offer a POSIX shell style experience for
 accessing external programs, without the significant risks posed by running
 ``os.system`` or enabling the system shell when using the ``subprocess`` module
-APIs, might provide an interface for running external programs similar to that
+APIs, which will provide an interface for running external programs similar to that
```

Comment:
```suggestion
APIs, which will provide an interface for running external programs inspired by that
```

(I realised we should soften the wording a bit here, as Julia's backticks offer full shell command pipeline support, which `shlex.sh` won't do. While you *could* build pipeline support atop template strings by also parsing the non-expression pieces, `shlex.sh` won't be going that far).